### PR TITLE
PD: optimize the description of label-property

### PR DESCRIPTION
**Pull request by niubell**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions))

### What is the related PR or file link(s)?

- This PR is translated from: None, just optimize some descriptions.
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -430,7 +430,7 @@ pd-server 相关配置项。
 
 ## label-property
 
-标签相关的配置项。
+标签相关的配置项，目前只支持 `reject-leader` 类型，后续会逐渐下掉该配置项，建议使用 [Placement Rule](https://docs.pingcap.com/zh/tidb/stable/configure-placement-rules#%E5%9C%BA%E6%99%AF%E4%BA%8C5-%E5%89%AF%E6%9C%AC%E6%8C%89-2-2-1-%E7%9A%84%E6%AF%94%E4%BE%8B%E6%94%BE%E7%BD%AE%E5%9C%A8-3-%E4%B8%AA%E6%95%B0%E6%8D%AE%E4%B8%AD%E5%BF%83%E4%B8%94%E7%AC%AC-3-%E4%B8%AA%E4%B8%AD%E5%BF%83%E4%B8%8D%E4%BA%A7%E7%94%9F-leader)。
```
Comment (qiancai) [inline pd-configuration-file.md:434]: 这个文档链接需要修改为 Markdown 链接，可以安装 [Copy Markdonw Link](https://chromewebstore.google.com/detail/copy-markdown-link/amekohfcgphhdbmnjklclhhmlklbpbnd) 这个 chrome 扩展一键得到文档的 Markdown 链接

Comment (niubell) [reply pd-configuration-file.md:434]: done

**Review by lhy1024 — APPROVED** [review]

File: pd-configuration-file.md [hunk 2]
```diff
@@ -430,7 +430,7 @@ pd-server 相关配置项。
 
 ## label-property
 
-标签相关的配置项。
+标签相关的配置项，目前只支持 `reject-leader` 类型，后续会逐渐下掉该配置项，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader)。
```
Comment (qiancai) [inline pd-configuration-file.md:433]:
```suggestion
标签相关的配置项，目前只支持 `reject-leader` 类型。该配置项计划将在未来版本中废弃，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader) 设置副本策略。
```

File: pd-configuration-file.md
Comment (qiancai) [inline pd-configuration-file.md:1]: 下面这两篇文档都提到了 “TiDB 5.2 及以上版本默认不支持 `label-property` 配置。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md)。”，请问是指 `label-property` 从 v5.2 开始不生效了吗


https://docs.pingcap.com/zh/tidb/dev/three-data-centers-in-two-cities-deployment
image

https://docs.pingcap.com/zh/tidb/dev/geo-distributed-deployment-topology
image

File: pd-configuration-file.md [hunk 3]
```diff
@@ -430,7 +430,7 @@ pd-server 相关配置项。
 
 ## label-property
 
-标签相关的配置项。
+标签相关的配置项，目前只支持 `reject-leader` 类型，该配置项已经从 v5.2 版本开始废弃，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader) 设置副本策略。
```
Comment (qiancai) [inline pd-configuration-file.md:433]:
```suggestion
标签相关的配置项，目前只支持 `reject-leader` 类型。该配置项已从 v5.2 开始废弃，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader) 设置副本策略。
```

**Review by qiancai — APPROVED** [review]

File: pd-configuration-file.md [hunk 4]
```diff
@@ -430,7 +430,7 @@ pd-server 相关配置项。
 
 ## label-property
 
-标签相关的配置项。
+标签相关的配置项，目前只支持 `reject-leader` 类型。该配置项已从 v5.2 开始废弃，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader) 设置副本策略。
```
Comment (qiancai) [inline pd-configuration-file.md:433]:
```suggestion
标签相关的配置项，只支持 `reject-leader` 类型。该配置项已从 v5.2 开始废弃，建议使用 [Placement Rules](/configure-placement-rules.md#场景二5-副本按-2-2-1-的比例放置在-3-个数据中心且第-3-个中心不产生-leader) 设置副本策略。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm
/approve

**Pull request merged**